.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### Prerequisites
+
+* [ ] Did you check past issues for similar issue(s) ?
+* [ ] Can you reproduce the issue with any of the files found in `sct_testing_data` ?
+* [ ] If you are using a development environment, is your environment updated ?
+
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:** [What you expect to happen]
+
+**Actual behavior:** [What actually happens]
+
+**Reproduces how often:** [What percentage of the time does it reproduce?]
+
+### Versions
+
+The version is found in `version.txt`.
```

Comment:
what I meant, is that if people are experiencing an issue from master, then if they report the version from version.txt, this will not be informative (because value would be "dev"). Instead, they should report the branch/SHA